C7/Util.cs

```diff
@@ -81,6 +81,14 @@ static public string Civ3PathFromRegistry() {
 	public static string FileExistsIgnoringCase(string exactCaseRoot, string ignoredCaseExtension) {
 		// First try the basic built-in File.Exists method since it's adequate in most cases.
 		string fullPath = System.IO.Path.Combine(exactCaseRoot, ignoredCaseExtension);
+
+		// In debug builds paths that are specified relative to the project directory
+		// will start with res://. We want to turn those into real paths, and then
+		// update our exactCaseRoot variable so that the search logic below works.
+		if (fullPath.Contains("res://")) {
```

Comment:
Would it not make more sense to check if exactCaseRoot contains "res://" and if so globalize it before combining it with the extension, rather than combining first and then extracting a substring? Or is "res://" maybe going to appear in the extension?

Reply:
Much simpler - thanks!

I thought I had tried that, but must have goofed something up in my earlier hacking.